docs/getting-started-guides/dcos.md

```diff
@@ -18,11 +18,6 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
-<!-- TAG RELEASE_LINK, added by the munger automatically -->
-<strong>
-The latest release of this document can be found
-[here](http://releases.k8s.io/release-1.1/docs/getting-started-guides/dcos.md).
-
```

Comment:
Why did this get touched? It looks like we removed the head of a munge tag without removing the tail.